packages/runtime/runtime-utils/src/dataStoreHelpers.ts

```diff
@@ -75,14 +73,14 @@ export function responseToException(response: IResponse, request: IRequest): Err
 	// As of 2025-08-20 the code seems to assume `response.value` is always a string.
 	// This type assertion just encodes that assumption as we move to stricter linting rules, but it might need to be revisited.
 	const message = response.value as string;
-	const errWithStack = generateErrorWithStack();
+	const errWithStack = "stack" in response ? response : generateErrorWithStack();
```

Comment:
Unlikely corner case:  If `response.stack` is set to `undefined` then the `stack` getter below will return `undefined` instead of falling back to a generated stack.  This is why the code above uses `??` after plugging in `error.stack`.  I guess we can't really cover this case anymore.

Probably fine.